src/OpenTelemetry/Trace/BatchingActivityProcessor.cs

```diff
@@ -202,22 +202,9 @@ public override async Task ForceFlushAsync(CancellationToken cancellationToken)
             OpenTelemetrySdkEventSource.Log.ForceFlushCompleted(this.currentQueueSize);
         }
 
-        /// <inheritdoc/>
-        public void Dispose()
+        protected override void Dispose(bool isDisposing)
```

Comment:
```suggestion
        protected override void Dispose(bool disposing)
```

Reply:
Follow this https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#dispose-and-disposebool.

Reply:
In this pr (https://github.com/open-telemetry/opentelemetry-dotnet/pull/955), i already solved this.